tools/create/package.json

```diff
@@ -0,0 +1,20 @@
+{
+  "name": "@lg-tools/create",
```

Comment:
i feel like if we're introducing this we should at least explore using turbo gen